bin/distServer.js

```diff
@@ -0,0 +1,42 @@
+import express from 'express';
+import favicon from 'serve-favicon';
+import path from 'path';
+import open from 'open';
+import parser from 'body-parser';
+import compression from 'compression';
+import userRoute from '../server/routes/users.routes';
```

Comment:
**Nitpic**
And instead of this endless import, you can have `index.js` in your routes directory for all of this.
For here, you simply import the `index.js` and use JS destructuring this way
```javascript
import { roleRoute, searchRoute, documentRoute, paginationRoute } from './routes'`
```

`searchRoute`, `paginationRoute` 👓